src/audio/pipeline/pipeline-graph.c

```diff
@@ -135,14 +135,16 @@ struct pipeline *pipeline_new(uint32_t pipeline_id, uint32_t priority, uint32_t
 	p->trigger.cmd = COMP_TRIGGER_NO_ACTION;
 	ret = memcpy_s(&p->tctx, sizeof(struct tr_ctx), &pipe_tr,
 		       sizeof(struct tr_ctx));
-	assert(!ret);
```

Comment:
@lgirdwood @lyakh I want to remove this memcpy_s() altogether. Do we really need this? p->ctx isnt used anywhere at all